akka-http/src/main/scala/akka/http/scaladsl/server/directives/SecurityDirectives.scala

```diff
@@ -314,7 +314,7 @@ object Credentials {
         new Credentials.Provided(token) {
           def verify(secret: String, hasher: String => String): Boolean = secret secure_== hasher(token)
         }
-      case Some(GenericHttpCredentials(scheme, token, params)) =>
+      case Some(_: HttpCredentials) =>
```

Comment:
Perhaps log the type of credentials, as we're no longer checking it's actually `GenericHttpCredentials` here?